tools/src/main/java/org/terracotta/utilities/io/Files.java

```diff
@@ -135,7 +135,8 @@ public final class Files {
 
   private static final Duration MINIMUM_OPERATION_REPEAT_DELAY = Duration.ofMillis(50);
   private static final int MAXIMUM_OPERATION_ATTEMPTS = 10;
-  private static final Duration DEFAULT_RENAME_TIME_LIMIT = Duration.ofMillis(2500L);
+  private static final long DEFAULT_OPERATION_TIME_LIMIT_MILLIS = 2500L;
```

Comment:
Why did we introduce this now?

Reply:
It's not being _introduced_ -- the name of the field is simply changed because its scope is broader than rename.

Reply:
Well ... it was split into a numeric constant and a derived constant used in the code for Javadoc purposes.